include/ModelWriter.h

```diff
@@ -2,17 +2,31 @@
 #define BIORBD_UTILS_WRITER_H
 
 #include "biorbdConfig.h"
-
+///
+/// \brief Namespace that holds the whole model
+///
 namespace biorbd {
 class Model;
 
+///
+/// \brief Namespace that holds the path
+///
 namespace utils {
 class Path;
 }
 
+///
+/// \brief Main class for writer
+///
 class BIORBD_API Writer
 {
 public:
+
+///
+/// \brief Writes the model
+/// \param model The model
+/// \param pathToWrite The path to write (folder in which to write)
```

Comment:
Remove folder part, since is it not just the folder, but the full path of the thing